go/libkb/globals.go

```diff
@@ -30,6 +30,14 @@ import (
 
 type ShutdownHook func() error
 
+type LoginHook interface {
+	OnLogin()
```

Comment:
should we change the login notifications we send to KBFS and Electron to use this?

Does "Login" mean "authenticate to session server"?